spotify_player/src/state/mod.rs

```diff
@@ -10,7 +10,7 @@ pub use model::*;
 pub use player::*;
 pub use ui::*;
 
-use crate::config;
+use crate::config::{self};
```

Comment:
This is a weird import. Is it different from `use crate::config`?

Reply:
Think this happened as I imported AppConfig directly at some stage e.g. `use crate::config::{self, AppConfig}`.

Reverted anyway 👍